microsoft-edge/extensions-chromium/publish/api/using-addons-publish-api.md

```diff
@@ -0,0 +1,210 @@
+---
+description: REST endpoints to automate publishing updates to add-ons that are submitted to the Microsoft Edge Add-ons website.
+title: Using the Microsoft Edge Add-ons Publish API
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 08/10/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, publish api, add-ons publish api
+---
+# Using the Microsoft Edge Add-ons Publish API
+
+This article, along with the [Add-ons Publish API Reference][AddonsPublishAPIRef], provides an overview of the proposed Microsoft Edge Add-ons Publish API.  We look forward to your suggestions and feedback on the proposed API contracts.  Please submit your feedback as an [Issue about the Add-ons API][GitHubMicrosoftDocsEdgeDeveloperNewIssue].
+
+The Microsoft Edge Add-ons Publish API provides a set of REST endpoints for programmatically publishing updates to add-ons submitted to the Microsoft Edge Add-ons website.  You can use these REST endpoints to automate the process of uploading and publishing add-ons into the Microsoft Edge Add-ons website.
+
+The tentative date for the availability of the APIs is December 2021.
+
+
+<!-- ====================================================================== -->
+## Terminology
+
+| Term | Definition |
+|---|---|
+| _operation_ | A REST operation such as GET or PUT.  There are several operations, defined at [Add-ons Publish API Reference][AddonsPublishAPIRef]. |
+| _operation ID_ | The ID of a REST operation. |
+| _package_ | The `.zip` package that contains the files for your Microsoft Edge Add-on. |
+| _product_ | A Microsoft Edge extension or theme.  Also referred to as a Microsoft Edge _Add-on_. |
+| _product ID_ | The product ID of the product whose draft needs to be published.  The product ID is a 32-character GUID that is associated with a product that is submitted to Partner Center.  For example: `d34f98f5-f9b7-42b1-bebb-98707202b21d`. |
+| _submission_ | An update that is being submitted to an existing product at Partner Center.  Every update to a product is a submission, regardless of whether the status is `In Draft`, `In Review`, or `In the Store` (published). |
+
+
+<!-- ====================================================================== -->
+## Before you begin
+
+To use the Microsoft Edge Add-ons Publish API, you need to enable the API for your project in the Microsoft Partner Center.
+
+1. Visit Microsoft Partner Center and login to the account from which you have already published an add-on.
+
+1. Under the **Microsoft Edge** program, a new page for **Publish APIs** appears.
```

Comment:
"new page" -- does the page appear now?  Or is the page planned to be added later?  Is it possible to take screenshots of Partner Center now, showing "Publish APIs"?  I try to avoid writing "new" in articles, b/c that goes stale immediately.  But we might need to convey that this is a plan, not impl'd yet.